hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -123,7 +123,7 @@ public void handleRound(@NonNull final HederaState state, @NonNull final Round r
         round.forEachEventTransaction((event, txn) -> {
             try {
                 handlePlatformTransaction(state, event, txn);
-            } catch (final Throwable e) {
+            } catch (final Exception e) {
```

Comment:
Again, not sure if this should be `Throwable`. Need to check all the places this was replaced

Reply:
Leaving this as is because even though we log at FATAL, we don't do a direct exit as the ones in `Hedera.java`.